### PR TITLE
Added LogOwl Adapter

### PR DESCRIPTION
**Pull request by vermakhushboo**
Added LogOwl Adapter.

Properties like path, line, stacktrace are picked from extras as Appwrite adds these properties in extras. How do we generalise this?

### CODE REVIEW

**Review by abnegate — COMMENTED** [review]
We could update the readme to show an example with LogOwl as well as add it to the supported adapter list 🙂 otherwise looks good to me 👍

**Review by Meldiron — CHANGES_REQUESTED** [review]
Great PR 🥳 Left my comments

File: tests/LoggerTest.php [hunk 1]
```diff
@@ -181,5 +184,11 @@ public function testAdapters()
         $logger = new Logger($adapter);
         $response = $logger->addLog($log);
         // self::assertEquals(202, $response);
+
+         // Test LogOwl
+         $adapter = new LogOwl(\getenv("TEST_LOGOWL_KEY"));
```
[the remark this answers is not in the document]
Comment (vermakhushboo) [reply tests/LoggerTest.php:189]: The test API key has been generated, need to set it.

[the remark this answers is not in the document]
Comment (Meldiron) [reply tests/LoggerTest.php:189]: API key set. Resolving the conversation

**Comment by Meldiron** [comment]
> We could update the readme to show an example with LogOwl as well as add it to the supported adapter list 🙂 otherwise looks good to me 👍

Took care of that. Also fixed existing README examples a little.

**Pull request closed without merging**